monitoring/uss_qualifier/scd/data_interfaces.py

```diff
@@ -16,16 +34,34 @@ class SCDTestIssue(ImplicitDict):
     details: str
     '''Human-readable description of the issue'''
 
-    def __init__(self, **kwargs):
-        super(SCDTestIssue, self).__init__(**kwargs)
-        if 'timestamp' not in kwargs:
-            self.timestamp = datetime.datetime.utcnow().isoformat()
+    queries: List[fetch.Query]
```

Comment:
How is the simulator going to populate this field?

Reply:
Thanks! I will remove this field.